bin/zkt

```diff
@@ -1,15 +1,23 @@
 #!/bin/bash
-zkt-raw | fzf --height 100% --no-info --no-multi \
-  --bind "ctrl-o:execute-silent[tmux send-keys -t \{left\} Escape :read Space ! Space echo Space && \
-          tmux send-keys -t \{left\} -l '\"\\'{2}'\"' && \
-          tmux send-keys -t \{left\} Enter]" \
-  --bind "ctrl-y:execute-silent(echo {2} | pbcopy),enter:execute[ \
-    ggrep -F --color=always -i {2} *.md -l | \
-      fzf --ansi --height 100% --preview-window=top:65% \
-        --bind 'enter:execute-silent$ \
-          tmux send-keys -t \{left\} Escape :e Space && \
-          tmux send-keys -t \{left\} -l \{} && \
-          tmux send-keys -t \{left\} Enter \
-        $' \
-        --preview 'bat --color always --language md --style plain \{}' \
-  ]"
+
+# load os detect script
+. os-detect
+
+case "${grep_command}" in
+  UNKNOWN*) echo "support only unix system";;
```

Comment:
Why not put this part in the os-detect file with an exit? 
It will not run anyway, and it would simplify the zkt{,-raw} scripts

Reply:
I just got the idea from you, just adding exit $? in the end of os-detect right?to be honest, my knowledge of shell scripts is lacking